setup.bat

```diff
@@ -0,0 +1,24 @@
+@echo off
+set "zipUrl=https://github.com/heysagnik/Linkees/archive/refs/heads/old.zip"
+set "zipFile=old.zip"
+set "extractDir=Linkees-old"
+
+echo Downloading %zipUrl%...
+curl -L -o "%zipFile%" "%zipUrl%"
+
+if %ERRORLEVEL% neq 0 (
+    echo Failed to download the zip file.
+    exit /b 1
+)
+
+echo Extracting %zipFile%...
+powershell -command "Expand-Archive -Path .\%zipFile% -DestinationPath '.'"
+if exist %zipFile% (
+    del %zipFile%
+)
+ren "Linkees-old" "Project"
+cd Project
+npm install
+

```

Comment:
This doesn't give user easy access to update the information on landing page. Moreover, why we are not setting up an app and starting it?

Reply:
would you like it to start the server?